tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -760,6 +789,69 @@ public void ViewAreProhibitedTest([Values(false, true)] bool async)
             }
         }
 
+        // NOTE: this test is not presented in the prose tests
+        [SkippableTheory]
+        [ParameterAttributeData]
+        public void UnsupportedPlatformsTests(
+            [Values("local", "aws", "azure", "gcp")] string kmsProvider,
+            [Values(false, true)] bool async)
+        {
+            RequireServer.Check().Supports(Feature.ClientSideEncryption);
+
+            using (var clientEncrypted = ConfigureClientEncrypted())
+            using (var clientEncryption = ConfigureClientEncryption(clientEncrypted.Wrapped as MongoClient))
+            {
+                Guid? dataKeyId = null;
+                var dataKeyOptions = CreateDataKeyOptions(kmsProvider);
+                var exception = Record.Exception(() => dataKeyId = CreateDataKey(clientEncryption, kmsProvider, dataKeyOptions, async));
+                AsserResult(exception);
+            }
+
+            void AsserResult(Exception ex)
+            {
+                var isLinux = RequirePlatform.GetCurrentOperatingSystem() == SupportedOperatingSystem.Linux;
+
+                switch (kmsProvider)
+                {
+                    // all crypto hooks fail on linux with .netstandard1.5
+                    case var _ when isLinux && ContainsCurrentTargetFramework(SupportedTargetFramework.NetCoreApp11):
+                        {
+                            var errorMessage = AssertExceptionTypesAndReturnErrorMessage<PlatformNotSupportedException>(ex);
+                            errorMessage.Should().Be($"Field-level encryption is not supported on Linux with .netstandard1.5.");
```

Comment:
errorMessage.Should().Be($"Field-level encryption is not supported on Linux with .NET Standard 1.5.");

Reply:
done